WifiART/src/userinterface/LcrView.java

```diff
@@ -60,6 +58,7 @@ private void setup() {
 
         JPanel lcrPanelTitlePanel = new JPanel();
         lcrPanelTitlePanel.setLayout(new FlowLayout(FlowLayout.LEFT));
+        lcrPanelTitlePanel.setPreferredSize(new Dimension(800, 10));
```

Comment:
Use defined constants to set these values.

Reply:
Fixed in the revision.